src/Autonomous.cpp

```diff
@@ -278,6 +278,13 @@ void Autonomous::autonomyIter()
 				plan_pose(2) += action(0);
 				plan_pose(0) += action(1) * cos(plan_pose(2));
 				plan_pose(1) += action(1) * sin(plan_pose(2));
+				transform_t tf_lidar_base = toTransform(pose);
+				transform_t tf_plan_pose = toTransform(plan_pose) * tf_lidar_base.inverse();
```

Comment:
Would it be more efficient to calculate the inverse of the transformation matrix outside of the for loop? Since tf_lidar_base has the same value every iteration it would save time to just compute it once before the loop.

Reply:
Fixed!